src/_components/skeletonLoader/SkeletonLoader.tsx

```diff
@@ -12,24 +12,29 @@ type TSkeletonProps = {
 };
 
 const SkeletonLoader: FC<TSkeletonProps> = ({ containerStyle, layout, dark = false }) => {
-  return (
-    <SkeletonContent
-      boneColor={theme.palette.neutral[dark ? '200' : '100']}
-      containerStyle={containerStyle}
-      duration={2000}
-      highlightColor={theme.palette.neutral[dark ? '100' : '0']}
-      isLoading
-      layout={layout.map(style => ({ ...styles.layout, ...style }))}
-    />
+  // As suggested in https://github.com/alexZajac/react-native-skeleton-content-nonexpo/issues/41#issuecomment-1138546643
+  const MemoizedSkeletonContent = useCallback(
+    () => (
+      <SkeletonContent
+        boneColor={theme.palette.neutral[dark ? '200' : '100']}
+        containerStyle={containerStyle}
+        duration={2000}
+        highlightColor={theme.palette.neutral[dark ? '100' : '0']}
+        isLoading
+        layout={layout.map(style => ({ ...styles.layout, ...style }))}
+      />
+    ),
+    [containerStyle, dark, layout],
   );
+
+  return <MemoizedSkeletonContent />;
 };
 
 export const styles = StyleSheet.create({
   layout: {
     alignSelf: 'flex-start',
-    borderRadius: 4
+    borderRadius: 4,
   },
 });
 
-
-export default SkeletonLoader;
\ No newline at end of file
+export default SkeletonLoader;
```

Comment:
default?

Reply:
Alle components worden als default exported en dan als named export in de `index.ts`